server.js

```diff
@@ -0,0 +1,23 @@
+const app = require('./app'),
```

Comment:
i like it, but why to create the server file and leave the app file? are those names proper now or its ambiguous?

Reply:
The idea is to separate the app definition from the server listening, so that the supertest lib can open and close the server without worrying about loose connections or multiple servers. If you don't agree with the naming or if you want to do it in another way, just tell me.